Unable to succesfully build

kind bug

**What happened**:
Attempting to run make on current master branch on a debian based machine fails.  I'm not sure why this would be, given that we're building in a Fedora container, but it seems that running make build on a debian based host is not working.  I typically run on Fedora 28 and 29, but was setting up a GKE instance using Stretch and ran into this issue.

Also as a data point, tried an 18.04 VM and had the same problem, tried a fresh Fedora 29 and was succesful.  Coincidence?

**What you expected to happen**:
make build should work as long as I have a working docker installation

**How to reproduce it (as minimally and precisely as possible)**:

On a debian based machine (stretch, and double checked on Ubuntu 18.04), checkout current master;
run `make build` and the build fails when setting up libvirt-devel-4.9.0

**Anything else we need to know?**:

Here's the output:

```
hack/dockerized "./hack/check.sh && KUBEVIRT_VERSION= ./hack/build-go.sh install " && ./hack/build-copy-artifacts.sh 
Sending build context to Docker daemon  6.144kB
Step 1/11 : FROM fedora:28
28: Pulling from library/fedora
f474e775f16f: Pull complete 
Digest: sha256:017eb7de7927da933a04a6c1ff59da0c41dcea194aaa6b5dd7148df286b92433
Status: Downloaded newer image for fedora:28
 ---> f0b9fdd4254d
Step 2/11 : ENV LIBVIRT_VERSION 4.9.0
 ---> Running in c20dc1fb4365
Removing intermediate container c20dc1fb4365
 ---> f9e32642aadf
Step 3/11 : RUN dnf install -y dnf-plugins-core &&     dnf copr enable -y @virtmaint-sig/virt-preview &&     dnf -y install libvirt-devel-${LIBVIRT_VERSION} make git mercurial sudo gcc findutils gradle rsync-daemon rsync qemu-img protobuf-compiler &&     dnf -y clean all
 ---> Running in e934679c3c57
Fedora 28 - x86_64 - Updates                    2.1 MB/s |  27 MB     00:13    
Fedora 28 - x86_64                              2.1 MB/s |  60 MB     00:28    
Last metadata expiration check: 0:00:14 ago on Wed Dec  5 06:30:15 2018.
Dependencies resolved.
================================================================================
 Package                       Arch        Version            Repository   Size
================================================================================
Installing:
 dnf-plugins-core              noarch      2.1.5-4.fc28       fedora       51 k
Installing dependencies:
 python3-dnf-plugins-core      noarch      2.1.5-4.fc28       fedora      141 k

Transaction Summary
================================================================================
Install  2 Packages

Total download size: 192 k
Installed size: 387 k
Downloading Packages:
(1/2): dnf-plugins-core-2.1.5-4.fc28.noarch.rpm 101 kB/s |  51 kB     00:00    
(2/2): python3-dnf-plugins-core-2.1.5-4.fc28.no 235 kB/s | 141 kB     00:00    
--------------------------------------------------------------------------------
Total                                           178 kB/s | 192 kB     00:01     
Running transaction check
Transaction check succeeded.
Running transaction test
Transaction test succeeded.
Running transaction
  Preparing        :                                                        1/1 
  Installing       : python3-dnf-plugins-core-2.1.5-4.fc28.noarch           1/2 
  Installing       : dnf-plugins-core-2.1.5-4.fc28.noarch                   2/2 
  Running scriptlet: dnf-plugins-core-2.1.5-4.fc28.noarch                   2/2 
  Verifying        : dnf-plugins-core-2.1.5-4.fc28.noarch                   1/2 
  Verifying        : python3-dnf-plugins-core-2.1.5-4.fc28.noarch           2/2 

Installed:
  dnf-plugins-core.noarch 2.1.5-4.fc28                                          
  python3-dnf-plugins-core.noarch 2.1.5-4.fc28                                  

Complete!
Repository successfully enabled.
Copr repo for virt-preview owned by @virtmaint- 349 kB/s | 117 kB     00:00    
Last metadata expiration check: 0:00:00 ago on Wed Dec  5 06:30:57 2018.
No match for argument: libvirt-devel-4.9.0
Error: Unable to find a match
The command '/bin/sh -c dnf install -y dnf-plugins-core &&     dnf copr enable -y @virtmaint-sig/virt-preview &&     dnf -y install libvirt-devel-${LIBVIRT_VERSION} make git mercurial sudo gcc findutils gradle rsync-daemon rsync qemu-img protobuf-compiler &&     dnf -y clean all' returned a non-zero code: 1
Makefile:16: recipe for target 'build' failed
make: *** [build] Error 1
```

--- a/hack/docker-builder/Dockerfile
+++ b/hack/docker-builder/Dockerfile
@@ -1,6 +1,6 @@
 FROM fedora:28
 
-ENV LIBVIRT_VERSION 4.9.0
+ENV LIBVIRT_VERSION 4.10.0
 
 RUN dnf install -y dnf-plugins-core && \
     dnf copr enable -y @virtmaint-sig/virt-preview && \

